src/App.css

```diff
@@ -52,3 +52,93 @@ nav.navbar .navbar-nav .nav-link.navbar-link-projects {
   margin-right: 10px;
 }
 
+
+/* Banner CSS */
+.banner {
+  margin-top: 0;
+  padding: 200px 0 100px 0;
+  background-image: url('./assets/img/banner.png');
+  background-position: top center;
+  background-size: auto;
+  background-repeat: no-repeat;
+  margin-top: 50px;
+}
+
+.banner .align-items-center img{
+  height: 600px;
+  animation: updown 1s linear infinite;
+  margin-top: 75px;
+}
+@keyframes updown {
+  0%, 30% { /* Start and quickly jump to the highest point */
+    transform: translateY(0);
+    animation-timing-function: cubic-bezier(0, 0, 1, 1); /* This makes the jump sudden */
+  }
+  40% { /* Peak of the jump */
+    transform: translateY(-70px); /* Adjust based on how high you want the jump */
+    animation-timing-function: ease-out; /* Starts fast, then slows down, simulating gravity on the way down */
+  }
+  100% {
+    transform: translateY(0);
+  }
+}
+
+.banner .tagline {
+  display: flex;
+  justify-content: center;
+  font-size: 30px;
+  font-family: pixelfont;
+  color: black;
+  padding-bottom: 20px;
+}
+
+.banner p {
+  font-size: 30px;
+  font-family: 'Arial';
+  color: black;
+}
+
+.banner h1 {
+  font-family: dogica;
+  color: black;
+  font-size: 40px;
+}
+
+@font-face {
```

Comment:
i think it is nicer to place these font declaration on top of the css file

Reply:
done